src/components/CodeView/index.tsx

```diff
@@ -70,6 +70,11 @@ export class CodeViewBase extends React.Component<Props> {
 
     return (
       <React.Fragment>
+        {selectedMessageMap &&
+          selectedMessageMap.global.length > 0 &&
+          isLineSelected(getCodeLineAnchorID(0), location) && (
+            <div id={getCodeLineAnchorID(0)} ref={_scrollToSelectedLine} />
```

Comment:
Hmm, I guess we need something more generic so that it works for `DiffView`, too:

https://github.com/mozilla/addons-code-manager/blob/76866e1634f73e5b3fc9629ef9095c81b77c9bdb/src/components/DiffView/index.tsx#L229

You could file a follow-up for that.

Reply:
Ah, yes, I missed that. I think when testing manually it looked like it was working because it still loads the file, and the global messages are at the top, but it doesn't scroll to the global messages. I'll file a follow-up.

Reply:
I opened #825